### ISSUE TITLE
Alert on Startup to Use Chrome

### ISSUE THREAD

**Issue by davidkircos** [quadratichq/quadratic#101]
Quadratic is only officially supported on Chrome and stable on Firefox. Safari is not well supported.

We should create a welcome message for users of Safari encouraging them to use Chrome.

**Comment by dumblob** [comment]
Just came across this alert using a Chromium browser. It can not be skipped.

I think it is dumb to limit the use to Chrome and Firefox. Actually this will most probably work in any modern webkit-based browser (Brave, Vivaldi, Opera, ...) and gecko-based browser.

I think this dialogue shall be just a warning with a "skip" button only to explain there is no formal support and limited functionality shall be expected. But it most not **disallow** the user to proceed. Web itself is defined as platform with "infinite" incremental graceful degradation approach.

Please make the dialogue "skippable" and allow others to enter without further constraining them.

Thanks!

**Comment by davidkircos** [comment]
You received this dialogue because your browser does not support WebAssembly. Quadratic can not work without web assembly, so the dialogue is not skippable. I will update the text on the dialogue to clarify the issue.

David

**Comment by dumblob** [comment]
Wow, so quick!

Thanks for the pointer. Of course the browser does fully support webassembly (it is the latest Chromium). But I have tested it with very strict settings in the [JShelter](https://jshelter.org/ ) extension without thinking this could have such severe effect.

Now I have set milder security settings in JShelter and it works like a charm. Thanks for debugging!

------
I do not know why, but I can not close this issue - the closing button is missing. Feel free to close it :wink:.
